### PR TITLE
299 refactoring renaming property cluster 04

### PR DESCRIPTION
**Pull request by alexander-schulze**
This pull request includes the renaming of seven object properties.

- 'measuresAgainstRule' to 'measures'
- 'offersService' to 'offers'
- 'originateFrom' to 'originatesFrom'
- 'hasPreviousRelatedProductionJob' to 'hasPreviousRelated'
- 'hasPublicationScheduleDate' to 'hasScheduleDate'
- 'hasRulePriority' to 'hasPriority'
- 'hasRuleReference' to 'hasReferencesRule'

**Reviewers:**
- @JuergenGrupp
- @aro-max

### CODE REVIEW

**Review by JuergenGrupp — APPROVED** [review]
All changes implemented as decided, thank you @alexander-schulze !
However, I would now prefer to rename ´hasRuleReference´ to ´hasRelatedRule´. This fits better with a set of other "hasRelated ..." properties. If you please apply this change and merge to dev, I do approve it right now.

**Comment by alexander-schulze** [comment]
@JuergenGrupp 
The property hasRuleReference has been renamed to hasRelatedRule.

**Comment by alexander-schulze** [comment]
@aro-max 
Looking forward to your feedback.

File: ontology/EBUCorePlus/ebucoreplus.owl [hunk 1]
```diff
@@ -3195,6 +3184,17 @@ ec:hasScene rdf:type owl:ObjectProperty ;
                        "hat Szene"@de .
 
 
+###  http://www.ebu.ch/metadata/ontologies/ebucoreplus#hasScheduleDate
+ec:hasScheduleDate rdf:type owl:ObjectProperty ;
+                   rdfs:range time:Instant ;
+                   dcterms:description "Pour exprimer spécifiquement la date d'horaire à laquelle un PublicationEvent est associé en particulier si l'heure de broacdast est après minuit. Par exemple, la date du planning serait le 29 mai et le programme est publié à 1 heure du matin le 30 mai, tout en restant associé dans le planning à la nuit du 29 mai."@fr ,
+                                       "To express specifically the schedule date to which a PublicationEvent is related in particular if the broacdast time is after midnight. For example, the schedule date would be May 29th and the programme is published at 1 am on May 30th, while still associated in the schedule with the night of May 29th."@en ,
```
Comment (aro-max) [inline ontology/EBUCorePlus/ebucoreplus.owl:3191]: typo : broadcast

**Review by aro-max — APPROVED** [review]
Thanks, some typos to correct.

**Comment by alexander-schulze** [comment]
@aro-max 
fixed the typo issues.

**Pull request merged**